docs: register f5xc-devcontainer plugin skills in governance CLAUDE.md

Add f5xc-devcontainer plugin's auto-activated skills (tool-catalog, self-awareness) to the source-of-truth CLAUDE.md that gets distributed to all 23 downstream repos via sync-managed-files.yml. Also adds a Container Awareness section documenting the plugin capabilities. This ensures the plugin references survive future governance syncs.

## CI Failure Report

**PR:** #250
**Run:** 23558198840
**Workflow:** Lint Code Base

### Failed Checks
NATURAL_LANGUAGE (textlint) — terminology rule

### Log Output
```
/github/workspace/CLAUDE.md
  49:51  error  Incorrect term: "CLI tool", use "command-line tool" instead    terminology
  99:27  error  Incorrect term: "CLI tools", use "command-line tools" instead  terminology

2 problems (2 errors, 0 warnings, 0 infos)
2 fixable problems.
```

### Action Required
Replace "CLI tool" with "command-line tool" and "CLI tools" with "command-line tools" in CLAUDE.md at lines 49 and 99, then push a new commit to the PR branch.

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -46,7 +46,7 @@ Invoke the relevant skill **before** starting work.
 | `superpowers:writing-plans` | Planning implementation work |
 | `superpowers:verification-before-completion` | Verifying work before marking complete |
 | `superpowers:finishing-a-development-branch` | Completing work on a branch |
-| `f5xc-devcontainer:tool-catalog` | Asking which CLI tool to use, how to use a tool, or any task requiring a tool decision |
+| `f5xc-devcontainer:tool-catalog` | Asking which command-line tool to use, how to use a tool, or any task requiring a tool decision |
 | `f5xc-devcontainer:self-awareness` | Asking "who are you", container identity, version, self-diagnosis, health check |
 
 **Activation rules** — invoke the matching skill automatically:
@@ -96,7 +96,7 @@ fix the code in the main session and re-delegate.
 When running inside the f5xc-salesdemos devcontainer, the
 `f5xc-devcontainer` plugin provides:
 
-- **Tool catalog** — 300+ CLI tools indexed by category with usage and auth info
+- **Tool catalog** — 300+ command-line tools indexed by category with usage and auth info
 - **Self-awareness** — live identity introspection via GitHub API
 - **Tool maintenance** — install/remove tools with automated GitHub issues
 - **Drift detection** — compare Dockerfile against tool catalog

